pkg/webhook/nodenetworkconfigurationpolicy/server.go

```diff
@@ -37,17 +37,11 @@ func Add(mgr manager.Manager) error {
 	// 1.- User changes nncp desiredState so it triggers deleteConditionsHook()
 	// 2.- Since we have delete the condition the status-mutate webhook get called and
 	//     there we set conditions to Unknown this final result will be updated.
-	server := webhookserver.New(mgr, webhookName, certificate.MutatingWebhook,
+	server := webhookserver.New(mgr.GetClient(), webhookName, certificate.MutatingWebhook,
 		webhookserver.WithPort(webhookPort),
 		webhookserver.WithHook("/nodenetworkconfigurationpolicies-mutate", deleteConditionsHook()),
 		webhookserver.WithHook("/nodenetworkconfigurationpolicies-status-mutate", setConditionsUnknownHook()),
 		webhookserver.WithHook("/nodenetworkconfigurationpolicies-timestamp-mutate", setTimestampAnnotationHook()),
 	)
-	return add(mgr, server)
-}
-
-// add adds a new Webhook to mgr with r as the webhook.Server
-func add(mgr manager.Manager, s manager.Runnable) error {
```

Comment:
I sthis change related?

Reply:
Yes, the webhookserver now has the function `Add` so we just delegate on that not calling mgr.Add anymore.